src/main/java/org/esbtools/gateway/resync/service/ResyncErrorMessages.java

```diff
@@ -0,0 +1,27 @@
+package org.esbtools.gateway.resync.service;
+
+import org.esbtools.gateway.resync.ResyncRequest;
+
+public final class ResyncErrorMessages {
+
+    public static final String INVALID_SYSTEM = "There is no resync configuration for this system: ";
+    public static final String INCOMPLETE_REQUEST = "One or more required values was not present: ";
+    public static final String SYSTEM_NOT_CONFIGURED = "One or more systems is not configured correctly: ";
+    public static final String RESYNC_FAILED = "There was a problem resyncing the selected message: %s";
```

Comment:
I think actually you might not need that annotation since you have the `@ExceptionHandler` methods which look like they might be more suitable to REST API's anyway.